sdks/agent-sdk/vitest.config.ts

```diff
@@ -3,7 +3,7 @@ import { defineConfig, type Plugin } from "vitest/config";
 
 export default defineConfig({
   // @see https://github.com/aleclarson/vite-tsconfig-paths/issues/176
-  plugins: [tsconfigPaths() as Plugin],
+  plugins: [tsconfigPaths()],
```

Comment:
if this assertion is no longer needed, mind removing the comment?

Reply:
Sure! ESLint removed it, I didn't even notice the comment. Thanks for mentioning it.